servo/connectors/kubernetes.py

```diff
@@ -2142,7 +2160,7 @@ async def create(
         ...
 
     @abc.abstractmethod
-    async def adjust(
```

Comment:
Why did we drop the async?

Reply:
The implementations and invocations are not async, I've just updated the abstract definition to align with that:

* https://github.com/opsani/servox/blob/bc974b3a10b9a49916d420f705d2c0d35fe3e23d/servo/connectors/kubernetes.py#L2397
* https://github.com/opsani/servox/blob/bc974b3a10b9a49916d420f705d2c0d35fe3e23d/servo/connectors/kubernetes.py#L2560
* https://github.com/opsani/servox/blob/bc974b3a10b9a49916d420f705d2c0d35fe3e23d/servo/connectors/kubernetes.py#L3144